javadoc/src/org/netbeans/modules/javadoc/highlighting/Highlighting.java

```diff
@@ -147,13 +154,34 @@ public void tokenHierarchyChanged(TokenHierarchyEvent evt) {
         if (seq.moveNext()) {
             int start = seq.offset();
             do {
+                String period = null;
+                int indexOfPeriod = -1;
+                for (String p : PERIODS) {
+                    int index = TokenUtilities.indexOf(seq.token().text(), p);
+                    if (index != -1) {
+                        if (indexOfPeriod == -1 || index < indexOfPeriod) {
+                            indexOfPeriod = index;
+                            period = p;
+                        }
+                    }
+                }
+
                 if (seq.token().id() == JavadocTokenId.DOT) {
                     if (seq.moveNext()) {
                         if (isWhiteSpace(seq.token())) {
                             return new int [] { start, seq.offset()};
                         }
                         seq.movePrevious();
                      }
+                } else if (period != null && indexOfPeriod != -1) {
+                    // NETBEANS-791
+                    int offset = TokenUtilities.indexOf(seq.token().text(), period) + 1;
```

Comment:
Isn't ``TokenUtilities.indexOf(seq.token().text(), period)`` the same value as ``indexOfPeriod`` ?

Reply:
You are right. Thank you for catching it.